src/GlobalStyle.ts

```diff
@@ -6,6 +6,10 @@ const GlobalStyle = createGlobalStyle`
 		font-family: ${getValueFromTheme('typography.fontFamily')};
 		color: ${themeColor('tint', 'level7')};
 	}
+
+	.modal--bulk {
```

Comment:
This should be scoped to `BulkWizard` where it is used using a styled-components approach.

Reply:
Yes, I've tried this, I couldn't get it to work.

Reply:
`export const ModalStyle = styled(Modal)`
This "works", the component works and is shown in the app but the styled styles are not applied to the element. 

`export const ModalStyle = styled(Modal)<ComponentProps<typeof Modal>>`
This doesn't work, errors about typings:

```
The inferred type of 'ModalStyle' cannot be named without a reference to '@amsterdam/bmi-component-library/node_modules/@amsterdam/asc-ui/lib/components/BackDrop/BackDropStyle'. This is likely not portable. A type annotation is necessary.ts(2742)
The inferred type of 'ModalStyle' cannot be named without a reference to '@amsterdam/bmi-component-library/node_modules/@amsterdam/asc-ui/lib/components/Portal'. This is likely not portable. A type annotation is necessary.ts(2742)
```
@fortunato